### PR TITLE
LLVM 20. KW 2022

### PR DESCRIPTION
**Pull request by jfrech**

### CODE REVIEW

File: shared/src/main/scala/effekt/context/Context.scala [hunk 1]
```diff
@@ -50,6 +52,10 @@ abstract class Context(val positions: Positions)
   // bring the context itself in scope
   implicit val context: Context = this
 
+  val DEFAULT_LLVM_VERSION: String = "12"
+  var LLVM_VERSION: String = DEFAULT_LLVM_VERSION
```
Comment (b-studios) [inline shared/src/main/scala/effekt/context/Context.scala:56]: You can add these options to `Config` and default them to "12".

File: tests-llvm/ancient/LLVMTests.scala [hunk 2]
```diff
@@ -0,0 +1 @@
+../jvm/src/test/scala/effekt/LLVMTests.scala
```
Comment (b-studios) [inline tests-llvm/ancient/LLVMTests.scala:1]: What is this file?

Comment (jfrech) [reply tests-llvm/ancient/LLVMTests.scala:1]: A symlink to the relevant testing code.

File: shared/src/main/scala/effekt/context/Context.scala (hunk 1 above)
Comment (jfrech) [reply shared/src/main/scala/effekt/context/Context.scala]: I hard-coded `"12"` in 0419025ca996340cf6e7370b6c4e43913f269edc for now.

**Comment by jfrech** [comment]
@b-studios @phischu Should we now delete the `singlestack` branch?

File: tests-llvm/ancient/LLVMTests.scala (hunk 2 above)
Comment (b-studios) [reply tests-llvm/ancient/LLVMTests.scala:1]: Usually, you would run `sbt testOnly effekt.LLVMTests` to only run those tests.

**Comment by b-studios** [comment]
> Should we now delete the singlestack branch?

If it only consisted of 50c234c and aad1553, then sure.

File: tests-llvm/ancient/LLVMTests.scala (hunk 2 above)
Comment (jfrech) [reply tests-llvm/ancient/LLVMTests.scala]: That is what we test in https://github.com/effekt-lang/effekt/blob/5d4058b8be40296016c8d635cf1157c041aef8fd/.github/workflows/ci.yml#L33; this symlink was a development relic. Removed in 5d4058b.

**Pull request merged**